roles/usegalaxy-eu.bashrc/files/galaxy_jwd.py

```diff
@@ -35,24 +35,40 @@ def main():
             <pg_host>:5432:*:<pg_user>:<pg_password>
 
         Example:
-            python galaxy_jwd.py get_jwd 12345678
-            python galaxy_jwd.py clean_jwds --dry_run True --days 30
+            get_jwd:
+                python galaxy_jwd.py get_jwd 12345678
+            clean_jwds:
+                Dry run: python galaxy_jwd.py clean_jwds --dry_run --days 5
+                No dry run: python galaxy_jwd.py clean_jwds --no_dry_run --days 5
         """,
     )
 
     # Parser for the get_jwd subcommand
-    get_jwd_parser = subparsers.add_parser("get_jwd", help="Get JWD path of a given Galaxy job id")
+    get_jwd_parser = subparsers.add_parser(
+        "get_jwd", help="Get JWD path of a given Galaxy job id"
+    )
     get_jwd_parser.add_argument(
         "job_id",
         help="Galaxy job id",
     )
 
     # Parser for the clean_jwds subcommand
-    clean_jwds_parser = subparsers.add_parser("clean_jwds", help="Clean JWD's of jobs failed within last X days")
-    clean_jwds_parser.add_argument(
+    clean_jwds_parser = subparsers.add_parser(
+        "clean_jwds", help="Clean JWD's of jobs failed within last X days"
+    )
+    dry_run_group = clean_jwds_parser.add_mutually_exclusive_group()
+    dry_run_group.add_argument(
         "--dry_run",
-        help="If True, do NOT delete JWD's; only print them (default: True)",
+        help="Dry run (default: True)",
+        dest="dry_run",
         default=True,
```

Comment:
If dry run is the default, why is there an option like that? You never need to specify it correct?

Reply:
I thought that the required parameter for python 3.6 was not allowed (we had this issue) but apparently it works, just tested it. So I can remove the default from `dry_run` so the user will be forced to choose either the `dry_run` or `no_dry_run`. In the current version, it is not the case so I added the default to one of them just so when no option is provided the default still be `dry_run`. I will update and push a new commit.

Reply:
Pushed a new commit https://github.com/usegalaxy-eu/infrastructure-playbook/pull/662/commits/beee7aa6f49d1465d29c877e5c92a6352edd0c83